components/esp_modem/examples/modem_tcp_client/components/extra_tcp_transports/include/mbedtls_wrap.hpp

```diff
@@ -18,10 +19,14 @@ class Tls {
 public:
     enum class is_server : bool {};
     enum class do_verify : bool {};
+    enum class session_op {
+        GET, SET, FREE, CHECK
+    };
 
     Tls();
     virtual ~Tls();
     bool init(is_server server, do_verify verify);
+    bool deinit();
```

Comment:
Function has no error path.
```suggestion
    void deinit();
```

Reply:
It currently doesn't, but I'd like to keep this option open. You probably remember our discussion about `init()` function of this wrapper, I wanted to keep it separate from the constructor, so we could use the object or have better control. The same applies for this `deinit()`.

Reply:
Ok, but cleaning up code usually doesn't have errors to return.